audio/portmidi/Portfile

```diff
@@ -16,30 +16,27 @@ license             MIT
 description         Free, cross platform, open-source, real-time MIDI I/O library.
 long_description    ${description}
 
-checksums           rmd160  6209405e5e9c4c41d8d474a84f1eba7d25babd56 \
-                    sha256  934f80e1b09762664d995e7ab5a9932033bc70639e8ceabead817183a54c60d0 \
-                    size    261654
+checksums           rmd160  381735e44525259ccb485431468357e92abfd4ac \
+                    sha256  64893e823ae146cabd3ad7f9a9a9c5332746abe7847c557b99b2577afa8a607c \
+                    size    263225
 
 # Building the Java interface and pmdefaults is disabled. They could be
 # added as separate ports, with a dependency on an appropriate openjdk
 # port, if there is interest.
 
-patchfiles          patch-CMakeLists.txt.diff \
-                    patch-pm_common-CMakeLists.txt.diff
+patchfiles          patch-CMakeLists.txt.diff
+
+# The author forgot to bump the version
+post-patch {
+    reinplace "s|2.0.3|${version}|g" ${worksrcpath}/CMakeLists.txt
+}
 
 if {[vercmp ${version} 2.0.4] < 0} {
     patchfiles-append \
                     patch-pm_mac-pmmacosxcm.c.diff \
                     patch-pm_test-CMakeLists.txt.diff
 }
```

Comment:
shouldn't this `if` statement just be removed?

Reply:
you now removed both if-statements... was that intentional? Is the second one not needed anymore, based on the comment one would think it is

Reply:
This is only required for version 2.0.3, as the missing file has now been added.

Reply:
I understand that the patchfile isn't needed anymore, but are you sure that the statement containing "os.major < 14" is redundant as well?

Reply:
Absolutely because I [fixed it myself](https://github.com/PortMidi/portmidi/commit/18cf5aa8312dbc38c1a03f9c1a53411b170fa61e).